README.md

```diff
@@ -54,18 +54,13 @@ It is also possible to use gulp-rev-replace without gulp-useref:
 var rev = require("gulp-rev");
 var revReplace = require("gulp-rev-replace");
 gulp.task("revision", ["dist:css", "dist:js"], function(){
-  return gulp.src(["dist/**/*.css", "dist/**/*.js"])
+  var manifest = gulp.src(["dist/**/*.css", "dist/**/*.js"])
     .pipe(rev())
     .pipe(gulp.dest(opt.distFolder))
     .pipe(rev.manifest())
-    .pipe(gulp.dest(opt.distFolder))
-})
-
-gulp.task("revreplace", ["revision"], function(){
-  var manifest = gulp.src("./" + opt.distFolder + "/rev-manifest.json");
 
   return gulp.src(opt.srcFolder + "/index.html")
-    .pipe(revReplace({manifest: manifest}))
+    .pipe(revReplace({manifest}))
     .pipe(gulp.dest(opt.distFolder));
```

Comment:
What version of `node` are you on? Shorthand (`{manifest}` instead of `{manifest: manifest}`) I believe was added in `iojs / node 4.x`.